README.md

```diff
@@ -21,46 +21,49 @@ SCALPEL can be installed and run using one of the following options:
 
 1. Clone the repository
 ```bash
-git clone https://github.com/p-CMRC-LAB/SCALPEL.git
-cd SCALPEL
+> git clone https://github.com/p-CMRC-LAB/SCALPEL.git
 ```
 
-2. Create the Conda environment
+2. Run SCALPEL using CONDA yml file
 ```bash
-conda env create -f requirements.yml
-conda activate scalpel_conda
+> nextflow run -resume SCALPEL/main.nf \
+  --sequencing chromium \
+  --samplesheet path/to/samplesheet.csv \
+  --transcriptome path/to/gencode.transcripts.fa \
+  --gtf path/to/gencode.annotation.gtf \
+  --ipdb path/to/mm10.polyA.track \
+  -with-conda SCALPEL/requirements.yml
 ```
 
-3. Run SCALPEL within the environment
+or create CONDA environment and activate
 ```bash
-nextflow run -resume main.nf \
+> conda env create --file SCALPEL/requirements.yml
+> conda activate scalpelEnv
+> nextflow run -resume main.nf \
   --sequencing chromium \
   --samplesheet path/to/samplesheet.csv \
   --transcriptome path/to/gencode.transcripts.fa \
   --gtf path/to/gencode.annotation.gtf \
-  --ipdb path/to/mm10.polyA.track \
-  --barcodes path/to/barcodes.csv \
-  --clusters path/to/clusters.txt
+  --ipdb path/to/mm10.polyA.track
 ```
 
 ---
 
 ### Option 2: Using Apptainer container
 
 You can download a prebuilt Apptainer container with all SCALPEL dependencies from the following link:  
-[Download SCALPEL Container](https://data.cyverse.org/dav-anon/iplant/home/franzx5/SCALPEL.container.sif)
+[Download SCALPEL Container](https://zenodo.org/records/15717636/files/SCALPEL.container.sif?download=1)
 
 1. Download the container and clone the repository
 ```bash
 wget https://data.cyverse.org/dav-anon/iplant/home/franzx5/SCALPEL.container.sif
 git clone https://github.com/p-CMRC-LAB/SCALPEL.git
-cd SCALPEL
 ```
 
 2. Run SCALPEL using the container
 ```bash
-nextflow run /path/to/SCALPEL/main.nf \
-  -with-apptainer /path/to/scalpel_container.sif \
+nextflow run SCALPEL/main.nf \
+  -with-apptainer SCALPEL.container.sif \
```

Comment:
```suggestion
  --with-apptainer SCALPEL.container.sif \
```

So maybe this is correct in the end.
Feel free to ignore my suggestions afte double-checking.
Would be a weird design decision of Nextflow though.